src/components/PrivateRoute.tsx

```diff
@@ -25,14 +39,14 @@ const PrivateRoute = ({ currentUser, initialized, render, ...props }) => {
     }
   } else {
     // App is still loading from local storage
-    return "Loading...";
+    return <div>Loading...</div>;
   }
 };
 
-const mapStateToProps = state => {
+const mapStateToProps = (state: State) => {
```

Comment:
As we've done elsewhere, we should probably rename this to `mapState` for consistency's sake.